framework/.classpath

```diff
@@ -12,5 +12,6 @@
 	<classpathentry combineaccessrules="false" exported="true" kind="src" path="/javacutil"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/annotation-file-utilities"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/scene-lib"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/javaparser-core"/>
```

Comment:
This change seems unrelated. Adding Eclipse support is a good idea, however I would call the project "stubparser-core" and add the corresponding configuration to the stubparser project.